src/diagnostics/define_diagnostics.jl

```diff
@@ -179,130 +181,276 @@ function define_diagnostics!(land_model)
-        units = "", # check
-        comments = "Net photosynthesis of a leaf.",
+        short_name = "fa",
+        long_name = "Cross Section",
+        standard_name = "cross_section",
+        units = "m s^-1",
+        comments = "Flux of water volume per m^2 of plant per second, multiplied by the area index (plant area/ground area).", 
         compute! = (out, Y, p, t) ->
-            compute_photosynthesis_net_leaf!(out, Y, p, t, land_model),
+            compute_cross_section!(out, Y, p, t, land_model),
+    )
+
+    # Root cross section
+    add_diagnostic_variable!(
+        short_name = "far",
+        long_name = "Root Cross Section",
+        standard_name = "Cross Section",
+        units = "m s^-1",
+        comments = "Flux of water volume per m^2 of root per second, multiplied by the area index (root area/ground area).", 
+        compute! = (out, Y, p, t) ->
+            compute_cross_section_roots!(out, Y, p, t, land_model),
+    )
+
+    # Leaf area index
+    add_diagnostic_variable!(
+        short_name = "lai",
+        long_name = "Leaf area Index",
+        standard_name = "leaf_area_index",
+        units = "m^2 m^-2",
+        comments = "The area index of leaves, expressed in surface area of leaves per surface area of ground.",
+        compute! = (out, Y, p, t) ->
+            compute_leaf_area_index!(out, Y, p, t, land_model),
     )
 
+    # Moisture stress factor
+    add_diagnostic_variable!(
+        short_name = "msf",
+        long_name = "Moisture Stress Factor",
+        standard_name = "moisture_stress_factor",
+        units = "",
+        comments = "Sensitivity of plants conductance to soil water content. Unitless",
+        compute! = (out, Y, p, t) ->
+            compute_moisture_stress_factor!(out, Y, p, t, land_model),
+    )
+
+    # Root area index
+    add_diagnostic_variable!(
+        short_name = "rai",
+        long_name = "Root area Index",
+        standard_name = "root_area_index",
+        units = "m^2 m^-2",
+        comments = "The area index of roots, expressed in surface area of roots per surface area of ground.",
+        compute! = (out, Y, p, t) ->
+            compute_root_area_index!(out, Y, p, t, land_model),
+    )
+
+    # Stem area index
+    add_diagnostic_variable!(
+        short_name = "sai",
+        long_name = "Stem area Index",
+        standard_name = "stem_area_index",
+        units = "m^2 m^-2",
+        comments = "The area index of stems, expressed in surface area of stems per surface area of ground.",
+        compute! = (out, Y, p, t) ->
+            compute_stem_area_index!(out, Y, p, t, land_model),
+    )
+
+    ### Canopy - Photosynthesis
+    # GPP - Gross Primary Productivity
     add_diagnostic_variable!(
         short_name = "gpp",
         long_name = "Gross Primary Productivity",
         standard_name = "gross_primary_productivity",
-        units = "",
-        comments = "Net photosynthesis of the canopy.",
+        units = "mol CO2 m^-2 s^-1",
```

Comment:
GPP, HR, AR should all be in the same units

Reply:
Definitely. I'll make sure they are all in the same units